lib/SchemaBuilder.ts

```diff
@@ -1,124 +1,125 @@
 
-import { Schema, XML, ExternalDocs, Response, BodyParameter } from 'swagger-schema-official';
```

Comment:
any meaningful changes here or just formatting?

Reply:
Formatting and changed this to a type import because ts-eslint-resolver was complaining